bolt_windows.go

```diff
@@ -67,6 +67,20 @@ func mmap(db *DB, sz int) error {
 	var sizelo, sizehi uint32
 
 	if !db.readOnly {
+		if db.MaxSize != 0 && db.MaxSize < sz {
```

Comment:
nit: It's more natural and readable to expect the subject `sz` to come first
```suggestion
		if db.MaxSize != 0 && sz > db.MaxSize {
```

Reply:
will change